ui/src/features/get-tests.js

```diff
@@ -43,6 +45,10 @@ class getTests extends React.Component {
       testToDelete: undefined,
       createTest: false,
       testForEdit: null,
+      testActionError: {
+        isError: false,
+        errorMessage: EMPTY_STRING,
+      },
```

Comment:
it will be nicer to use one variable for that - something like renderTestError 
and its value would be the error text if error accrued and undefined/null if it didn't.

Reply:
done!